### ISSUE TITLE
Navidrome lists genre view as supported, but it is not.

### ISSUE THREAD

**Issue by Christoph-Wagner** [basings/selfhosted-music-overview#170]
Navidrome only supports filtering/searching by genre
![image](https://user-images.githubusercontent.com/1609329/210201422-2ec2bed5-00f9-4714-9376-688e36af0edc.png)
 but has no genre view the way I understand it, for example how Jellyfin does it, showing a view of available genres: 
![image](https://user-images.githubusercontent.com/1609329/210201439-55363f90-6e5a-4084-a910-3d33549a5c34.png)

I’d have filled a PR, but I’m not sure if I understand the meaning of "Genre View" correctly ;)

**Comment by basings** [comment]
I think it's hard to say that navidrome doesn't have genre view since you can only apply the filter genre and it'll list all albums that have a specific tag. I agree that it doesn't have a dedicated genre button but the result is the same, isn't it?

**Comment by Christoph-Wagner** [comment]
While the Jellyfin view is suboptimal, at least it gives you some information (via album covers) of the genre members. If a dropdown as genres for a search is enough, wouldn’t every app have a genre view? This is where I decided to check, and no, they actually don’t even allow you to do a search by genre. I guess I overestimated the capabilities of modern ultra-simplified music player interfaces. Considering that, I concede the point ;)
